featurebyte/feast/patch.py

```diff
@@ -158,6 +185,4 @@ def get_transformed_features_df(
             # Long name must be in dataframe.
             rename_columns[long_name] = short_name
 
-    # Cleanup extra columns used for transformation
-    df_with_features.drop(columns=columns_to_cleanup, inplace=True)
```

Comment:
This statements takes about 5-10% runtime of the  get online features for a feature list with 306 features.